src/node/internal/internal_process.ts

```diff
@@ -6,24 +6,180 @@
 // is only an approximation of process.nextTick semantics. Hopefully it's good
 // enough because we really do not want to implement Node.js' idea of a nextTick
 // queue.
-/* eslint-disable */
 
 import { validateObject } from 'node-internal:validators';
 
-import { ERR_INVALID_ARG_VALUE } from 'node-internal:internal_errors';
+import {
+  ERR_INVALID_ARG_VALUE,
+  NodeError,
+} from 'node-internal:internal_errors';
 
-import { default as utilImpl } from 'node-internal:util';
+import { default as processImpl } from 'node-internal:process';
+import {
+  _initialized as eventsInitialized,
+  default as EventEmitter,
+} from 'node-internal:events';
 
-export function nextTick(cb: Function, ...args: unknown[]) {
+export const versions = processImpl.versions;
+
+export const version = `v${processImpl.versions.node}`;
+
+export const title = 'workerd';
+
+export const argv = ['node'];
+
+export const argv0 = 'node';
+
+export const execArgv = [];
+
+export const arch = 'x64';
+
+export const platform = processImpl.processPlatform;
+
+export const release = {
+  name: 'node',
+  sourceUrl: `https://nodejs.org/download/release/v${version}/node-v${version}.tar.gz`,
+  headersUrl: `https://nodejs.org/download/release/v${version}/node-v${version}-headers.tar.gz`,
```

Comment:
I don't think we should expose these properties.

Reply:
It's just that both Bun and Deno implement this, and I was wondering if there are compat paths that use `process.release.name === 'node'` as a compat gate.

Perhaps we just make the URLs undefined to allow compat checks that happen to look like that?

The issue is that the closer we get to Node.js compat, the harder it is to exclude objects which are expected.

Reply:
Removed.